gpii/node_modules/transformer/test/TransformerTests.js

```diff
@@ -1551,3 +1551,79 @@ jqUnit.test("gpii.transformer.hasSupportedSettings() tests", function () {
         jqUnit[oneTest.expected ? "assertTrue" : "assertFalse"]("gpii.transformer.hasSupportedSettings() test - " + oneTest.name, result);
     });
 });
+
+var jsonArrayToTaggedArrayPayload = {
+    "transform": {
+        "taggedArray": {
+            "transform": {
+                "type": "gpii.transformer.jsonArrayToTaggedArray",
+                "input": [
+                    "Buttons",
+                    "Lists",
+                    "Menus"
+                ],
+                "tag": "ReadItem",
+                "key": "$t",
+                "elems": [
+                    "Buttons",
+                    "Lists",
+                    "Menus",
+                    "Text",
+                    "Other"
+                ],
+                "hasElemVal": {
+                    "isTrue": "true",
+                    "isFalse": "false"
+                }
+            }
+        }
+    },
+    "model": {},
+    "expected": {
+        "taggedArray": [
+            {
+                "$t": "Buttons",
+                "ReadItem": "true"
+            },
+            {
+                "$t": "Lists",
+                "ReadItem": "true"
+            },
+            {
+                "$t": "Menus",
+                "ReadItem": "true"
+            },
+            {
+                "$t": "Text",
+                "ReadItem": "false"
+            },
+            {
+                "$t": "Other",
+                "ReadItem": "false"
+            }
+        ]
+    },
+    "invExpected": [
```

Comment:
The listed inverse is incorrect - it should be the same as the input. Please supply more than one test case, including ones in which there are i) all tags "false", ii) all tags "true" and iii) a mixture as here. Use a facility similar to that in https://github.com/fluid-project/infusion/blob/master/tests/framework-tests/core/js/ModelTransformationTests.js#L91 in order to assert that the inversion is good in a mechanical way.

Reply:
Same reason as described in the comment above, now fixed, and the requested tests cases have been included!